package.json

```diff
@@ -22,14 +22,15 @@
   ],
   "scripts": {
     "preprocess": "node tasks/preprocess.js",
+    "precommit": "lint-staged",
     "bundle": "node tasks/bundle.js",
     "header": "node tasks/header.js",
     "stats": "node tasks/stats.js",
     "build": "npm run preprocess && npm run bundle && npm run header && npm run stats",
     "cibuild": "npm run preprocess && node tasks/cibundle.js",
     "watch": "node tasks/watch.js",
-    "lint": "eslint --version && eslint .",
```

Comment:
Agreed. Asking on the gitter channel just in case since I wasn't able to find a better answer, but I think that alone might be enough for me. Seems some people run it through both, but that seems a bit annoying (though not necessarily slow if you prettier + eslint *only* staged files).